packages/trader/src/App/Components/Elements/market-countdown-timer.jsx

```diff
@@ -134,10 +134,10 @@ const MarketCountdownTimer = ({ active_symbols, is_main_page, setIsTimerLoading,
         const opening_time_moment_obj = moment(opening_time, 'HH:mm');
         const opening_time_hour = opening_time_moment_obj.format('HH');
         const opening_time_min = opening_time_moment_obj.format('mm');
-        const formatted_opening_time =
-            Number(opening_time_hour) > 11
-                ? `${Number(opening_time_hour) % 12}:${opening_time_min} pm`
-                : `${Number(opening_time_hour)}:${opening_time_min} am`;
+        const formatted_opening_time = `${
```

Comment:
Can we refactor this entire logic into its own function? preferably in `@deriv/shared`?